code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -274,6 +274,10 @@
 /obj/item/gun/magic/tentacle/shoot_with_empty_chamber(mob/living/user as mob|obj)
 	to_chat(user, span_warning("The [name] is not ready yet."))
 
+/obj/item/gun/magic/tentacle/attack_secondary(mob/living/victim, mob/living/user, params)
+	..()
+	return fire_gun(victim, user)
```

Comment:
Did this work by the way? I think attack_secondary has specific return values it expects, and you also call the parent proc so wont this just hold someone up and then immediately shoot them?

Correct me if I’m wrong, I can’t check right now

Reply:
From testing with combat mode on it just shoot the tentacle instead. I can retest it once I'm home

Reply:
I think it might act weird then in non-combat mode